terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java

```diff
@@ -0,0 +1,207 @@
+            fail(); // FAIL when exception is thrown
+        }
+    }
+
+    @Test
+    public void testMergePdfDocument() {
+        try {
+            reservationReportPdfStamperView.mergePdfDocument(model, stamper,
+                    request, response);
+
+            AcroFields form = stamper.getAcroFields();
+            assertThat(form.getField("referenceName"), is(
+                    "TERASOLUNA TRAVEL CUSTOMER CENTER"));
+            assertThat(form.getField("referenceEmail"), is(
+                    "customer@example.com"));
+            assertThat(form.getField("referenceTel"), is("01-2345-6789"));
+            assertThat(form.getField("paymentMethod"), is("Bank Transfer"));
+            assertThat(form.getField("paymentAccount"), is(
+                    "Current Account：12345678"));
+            assertThat(form.getField("childCount"), is("3"));
+            assertThat(form.getField("tourName"), is(
+                    "【おすすめ】Terasolunaツアー(6泊7日)"));
+            assertThat(form.getField("accomName"), is("TERASOLUNAホテル第一荘"));
+            assertThat(form.getField("customerKana"), is("キムラ　タロウ"));
+            assertThat(form.getField("customerTel"), is("111-1111-1111"));
+            assertThat(form.getField("adultUnitPrice"), is("75000"));
+            assertThat(form.getField("reservedDay"), is("2019/02/21"));
+            assertThat(form.getField("conductor"), is("Yes"));
+            assertThat(form.getField("tourAbs"), is(
+                    "そこは別天地、静寂と湯けむりの待つ宿へ… 詳しい情報はお取り合わせをお願い致します。"));
+            assertThat(form.getField("customerAdd"), is("千葉県八千代市上高野"));
+            assertThat(form.getField("customerJob"), is("プログラマ"));
+            assertThat(form.getField("tourDays"), is("7"));
+            assertThat(form.getField("depDay"), is("2019/03/31"));
+            assertThat(form.getField("customerName"), is("木村　太郎"));
+            assertThat(form.getField("childUnitPrice"), is("37500"));
+            assertThat(form.getField("depName"), is("北海道"));
+            assertThat(form.getField("customerBirth"), is("1975/01/05"));
+            assertThat(form.getField("arrName"), is("北海道"));
+            assertThat(form.getField("customerMail"), is("tarou@example.com"));
+            assertThat(form.getField("adultCount"), is("5"));
+            assertThat(form.getField("customerCode"), is("00000001"));
+            assertThat(form.getField("reserveNo"), is("00000001"));
+            assertThat(form.getField("remarks"), is("特になし"));
+            assertThat(form.getField("accomTel"), is("018-123-4567"));
+            assertThat(form.getField("customerPost"), is("276-0022"));
+            assertThat(form.getField("printDay"), is("2019/03/06"));
+            assertThat(form.getField("adultPrice"), is("375000"));
+            assertThat(form.getField("childPrice"), is("112500"));
+            assertThat(form.getField("sumPrice"), is("487500"));
+            assertThat(form.getField("paymentTimeLimit"), is("2019/03/24"));
```

Comment:
- [ ] You should check whether or not unnecessary fields are registered.

Reply:
fd313eba7b0eac1ceb70eee785765dd3b68fb95b

I fixed in above commit .

Reply:
If you assert size of fields, I recommend to do it first.

Reply:
2c950321d95a99d81e6f6318bce3d7095bee7d5c

I fixed in above commit .